debug/process/debugger.go

```diff
@@ -25,7 +27,7 @@ type processDebugger struct {
 	goRoutinesDumpHandler   func()
```

Comment:
`goRoutinesDumpHandler` and `logChangeHandler` can be removed and call the functions directly?

Reply:
no, can not properly test the component otherwise